src/app/+search-page/search-filters/search-filter/search-facet-filter/search-facet-filter.component.ts

```diff
@@ -126,7 +126,7 @@ export class SearchFacetFilterComponent implements OnInit, OnDestroy {
         this.animationState = 'ready';
         this.filterValues$.next(rd);
       }));
-      this.subs.push(newValues$.pipe(first()).subscribe((rd) => {
+      this.subs.push(newValues$.pipe(take(1)).subscribe((rd) => {
```

Comment:
I'm not sure I understand the reason why we are switching from `first()` to `take(1)`?  If this is necessary, we should also remove `first` from the `import` statement above

Reply:
`first()` and `take(1)` are nearly identical, the key difference is that `first()` will expect to get at least one value, and will throw an EmptyError if the observable completes without firing one. `take(1)` will simply never execute the subscribe if there are no values. @Atmire-Kristof was getting these EmptyErrors in certain cases, and that's why we replaced them.

We'll remove the import for first